php/templates/includes/community-containers.twig

```diff
@@ -0,0 +1,38 @@
+<h2>Community Containers</h2>
+<p>In this section you can enable or disable optional Community Containers that are not included by default in the main installation. These containers are provided by the community and can be useful for various purposes.</p>
+<p><strong>Caution: </strong>Community Containers are not officially component and it maintained by the community. Some containers may not be compatible with your system, may not work as expected or may discontinue. Use them at your own risk.</p>
```

Comment:
Maybe we could add a hint here to read first through the documentation of each container before adding any...

Reply:
done